### PR TITLE
Converted workflows to Orquesta

### LINKED ISSUE #7: Convert mistral-v2 workflows to orquesta workflows
Master branch is currently failing because of the use of mistral-v2 workflow.

https://app.circleci.com/pipelines/github/StackStorm-Exchange/stackstorm-duo/28/workflows/69e0c1fa-17e1-452a-8772-b5db33c136f8/jobs/229

```
2020-08-31 13:32:49,302 WARNING [-] Failed to register actions: Failed to register action "/tmp/packs/duo/actions/auth_chatops.yaml" from pack "duo": RunnerType mistral-v2 is not found. If you are using old and deprecated runner name, you need to switch to a new one. For more information, please see https://docs.stackstorm.com/upgrade_notes.html#st2-v0-9
Traceback (most recent call last):
  File "/tmp/st2/st2common/bin/st2-register-content", line 22, in <module>
    sys.exit(content_loader.main(sys.argv[1:]))
  File "/tmp/st2/st2common/st2common/content/bootstrap.py", line 408, in main
    register_content()
  File "/tmp/st2/st2common/st2common/content/bootstrap.py", line 358, in register_content
    register_actions()
  File "/tmp/st2/st2common/st2common/content/bootstrap.py", line 229, in register_actions
    raise e
ValueError: Failed to register action "/tmp/packs/duo/actions/auth_chatops.yaml" from pack "duo": RunnerType mistral-v2 is not found. If you are using old and deprecated runner name, you need to switch to a new one. For more information, please see https://docs.stackstorm.com/upgrade_notes.html#st2-v0-9
Registering resources for pack duo failed
make: *** [Makefile:192: .packs-resource-register] Error 1
```

Convert these to orquesta.

### PR DESCRIPTION
**Pull request by nmaludy**
Closes #7

### CODE REVIEW

File: actions/workflows/auth_chatops.yaml [hunk 1]
```diff
@@ -1,44 +1,42 @@
...
+  - user
+  - channel
+  - hint
+
+output:
+  - auth: "{{ ctx().auth }}"
+  - message: "{{ ctx().message }}"
+
+tasks:
+  duo_auth:
```
Comment (amanda11) [inline actions/workflows/auth_chatops.yaml:16]: The comments with the # of the placement of the boxes has been lost in the translation - don't know if its preferable to keep them - it's obviously doesn't stop it working - so not sure if worth keeping or not.  So leaving as an observation but not blocking merge.

**Review by amanda11 — APPROVED** [review]

File: actions/workflows/auth_chatops.yaml (hunk 1 above)
Comment (blag) [reply actions/workflows/auth_chatops.yaml:16]: From my previous experiments with this, I don't think those comments are parsed and stored by ST2, then they aren't passed to the web UI, and therefore are "ignored" by the web UI. So I don't think removing them will matter.

**Review by blag — APPROVED** [review]
LGTM

File: actions/workflows/auth_chatops.yaml (hunk 1 above)
Comment (amanda11) [reply actions/workflows/auth_chatops.yaml:16]: The  # comments are used by the web UI for Orquesta workflows (in the EWC workflow designer they definitely affect how it shows the boxes), however as the placement of them in an action-chain might be different to the orquesta designer then it probably doesn't matter that they are lost, as the two views might differ anyway.  So the numbers might be the wrong values, so in hind-sight probably best to remove.

**Pull request merged**